hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieROTablePathFilter.java

```diff
@@ -148,8 +148,9 @@ public boolean accept(Path path) {
 
       // Skip all files that are descendants of .hoodie in its path.
       String filePath = path.toString();
-      if (filePath.contains("/" + HoodieTableMetaClient.METAFOLDER_NAME + "/")
-          || filePath.endsWith("/" + HoodieTableMetaClient.METAFOLDER_NAME)) {
+      if ((filePath.contains("/" + HoodieTableMetaClient.METAFOLDER_NAME + "/")
+          || filePath.endsWith("/" + HoodieTableMetaClient.METAFOLDER_NAME))
+          && !filePath.contains("/" + HoodieTableMetaClient.ERROR_TABLE_FOLDER_NAME)) {
```

Comment:
can you clarify a bit about why exclude error table from this skipping? thought everything under `.hoodie/` should be skipped here?

Reply:
Yes, the error table is another hudi table, if it is not skipped, it will affect the query of the normal data table